### PR TITLE
Fixes #7520 Refactor/update description dropdown multiselect map

### LINKED ISSUE #7520: [refactor]: Update descriptions(tooltips) for dropdown, multiselect and map components
### Summary
Descriptions(tooltips) for components need to be updated.

### How to reproduce the issue?

1. Open the ToolJet App-Builder
2. Hover over the components in the Components library on the right
3. You can see descriptions(tooltips) for each component on hover

### Screenshots 

Table component description(tooltip) example:

table-en

table-widget-config

image

### Any possible solutions?

Replace the current labels in the `widgetConfig.js` and `en.json` files according to the below table.

| Name             | Current Description                  | New Description       |
|:-----------------|:---------------------------------|:----------------------|
| Dropdown         | Select one value from options    | Single item selector     |
| Multiselect      | Select multiple values from options | Multiple item selector     |
| Map              | Display Google Maps              | Display map locations            |

### If the bug is confirmed, would you be willing to submit a PR?
No

### Code of Conduct

- [ ] I agree to follow the [ToolJet Code of Conduct](https://docs.tooljet.com/docs/contributing-guide/code-of-conduct)

To seek assistance or resolve issues, we invite you to join our active community of ToolJet developers on [Slack](https://tooljet.com/slack).

### PR DESCRIPTION
**Pull request by Amruta24818**
fixes #7520 
Updated description for dropdown, multiselect and map components.

### CODE REVIEW

**Comment by Amruta24818** [comment]
Hey @karan-rathod-316 please review this PR.

File: server/.version [hunk 1]
```diff
@@ -1 +1,2 @@
 2.19.0
+
```
Comment (tejakummarikuntla) [inline server/.version:2]: Could you please remove this line you added?

**Review by tejakummarikuntla — CHANGES_REQUESTED** [review]
Hey @Amruta24818 , could you address the requested changes?

File: server/.version (hunk 1 above)
Comment (Amruta24818) [reply server/.version]: Removed line added by me

**Comment by Amruta24818** [comment]
@tejakummarikuntla Can you please review this PR?

**Review by tejakummarikuntla — APPROVED** [review]

**Pull request merged**